inference/hf-text-generation-inference/README.md

```diff
@@ -39,10 +39,14 @@ curl 127.0.0.1:8080/generate_stream \
     -H 'Content-Type: application/json'
 ```
 
-Further information can be found in the documentation of the [hf text-generation-inference](https://github.com/huggingface/text-generation-inference) solution.
-
-
-
+When performing inference on the chat-based model, ensure that the input prompt adheres to a specific structure that includes tags such as `<s>..</s>`, `[INST] .. [/INST]`, and optional `<<SYS>>` for system prompts. The structure of the input prompt should be as follows:
```

Comment:
Could you include a link to this blog post to give the reader a bit more context?
https://huggingface.co/blog/llama2#how-to-prompt-llama-2